### DISCUSSION TITLE
Hide colluns on gui

### DISCUSSION THREAD

**Discussion by leoborlot** [beekeeper-studio/beekeeper-studio#982] [Ideas]
Hi guys. I'm in love with beekeeper. 

There's a feature then I need to turn more easy to my workers. I looking for a feature to hide some Collins on show tables or views. I don't find the resource. If exist, please show where. If not, please. Add this on the soon versions. 

Very very thanks for this project. It solve many problems here.

**Comment by rathboma** [comment]
Hi! Unfortunately this isn't possible yet, but we'll hopefully be adding it soon!
